libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/CompilerArgumentsContributor.kt

```diff
@@ -44,7 +44,7 @@ internal open class AbstractKotlinCompileArgumentsContributor<T : CommonCompiler
 
     protected val logger by taskProvider.map { it.logger }
 
-    private val isMultiplatform by taskProvider.map { it.isMultiplatform }
+    private val isMultiplatform = taskProvider.map { it.isMultiplatform }.get()
```

Comment:
💄 Don't `.get()` the provider for uniformity with the other fields in this class